### DISCUSSION TITLE
Custom local map

### DISCUSSION THREAD

**Discussion by gilles-sxb** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#364] [Q&A] [answered]
Dear all,

I have a local png file that i use for the local map. But if i want to replace them by another one, i still see the old png file. I can delete the file, modify the png file it's still the same first image. Is there somewhere a cache?

/Gilles

**Comment by Henk72** [comment]
What is the file location of the png file? Maybe try a reboot.

**Comment by PiotrMachowski** [comment, marked as answer]
Hmm, somehow I have missed this question. Cache is built-in in your browser, you just have to clear it or change filename of map image.
